src/lib/classes/digital-asset.ts

```diff
@@ -20,14 +22,27 @@ export class DigitalAsset {
     this.signer = new NonceManager(options.signer);
   }
 
-  deployLSP7DigitalAssetReactive(digitalAssetDeploymentOptions: LSP7DigitalAssetDeploymentOptions) {
-    const digitalAsset$ = lsp7DigitalAssetDeployment$(this.signer, digitalAssetDeploymentOptions);
-
+  deployLSP7DigitalAssetReactive(
+    digitalAssetDeploymentOptions: LSP7DigitalAssetDeploymentOptions,
+    contractDeploymentOptions?: ContractDeploymentOptions
+  ) {
+    const digitalAsset$ = lsp7DigitalAssetDeployment$(
+      this.signer,
+      digitalAssetDeploymentOptions,
+      contractDeploymentOptions?.libAddress ??
+        versions[this.options.chainId]?.baseContracts?.LSP7DigitalAsset['0.0.1']
```

Comment:
`'0.0.1'` is the contract version number?
Would it make sense to put this at the top of the file like CONTRACT_VERSION = '0.0.1' or in a const/globals file ?